packages/ui-components/src/components/DataGridRow/DataGridRow.component.tsx

```diff
@@ -4,32 +4,14 @@
  */
 
 import React, { forwardRef } from "react"
+import "./data-grid-row.css"
 
-const rowBaseStyle = `
-	jn:contents
-`
-// const rowSelectedStyle = `
-// 	jn:bg-theme-datagridrow-selected
-// `
 export const DataGridRow = forwardRef<HTMLDivElement, DataGridRowProps>(
-  ({ /* selected, disabled,*/ className = "", children, /*onChange,*/ ...props }, ref) => {
-    // const dataGridContext = useDataGridContext() || {}
-    // const selectable = dataGridContext.selectable
+  ({ hoverable = false, className = "", children, ...props }, ref) => {
+    const finalStyles = `${hoverable ? "juno-datagrid-group" : ""} juno-datagrid-row ${className}`
 
-    // const [isSelected, setIsSelected] = useState(false)
-    // useEffect( () => {
-    // 	setIsSelected(selected)
-    // }, [selected])
-
-    // const toggleSelected = (event) => {
-    // 	setIsSelected(!isSelected)
-    // 	onChange(event)
-    // }
-
-    //  ${ selectable && isSelected ? rowSelectedStyle : '' }
     return (
-      <div className={`juno-datagrid-row ${rowBaseStyle} ${className}`} role="row" ref={ref} {...props}>
-        {/* { selectable ? <DataGridCheckboxCell selected={selected} disabled={disabled} onChange={toggleSelected} /> : null } */}
+      <div className={finalStyles} role="row" ref={ref} {...props}>
```

Comment:
Please bring the class name string back here instead of offloading it into a const.